src/IOBuf.h

```diff
@@ -64,6 +64,7 @@ class IOBuf {
   }
 
   void TrimEnd(size_t amount) { length_ -= amount; }
+  void SetLength(size_t amount) { length_ = amount; }
```

Comment:
I think TrimEnd() and Length() could be used to achieve the same behavior

Reply:
ok lemme check, i remember trying it, but will check again